po/de.po

```diff
@@ -77,9 +80,13 @@ msgid ""
 "the edge of the forest and find your enemy. You attack, causing\n"
 "damage of 5000 hp. You feel satisfied.\n"
 msgstr ""
+"\n"
+"Your single slash has been ignored. You walk to the market\n"
```

Comment:
Untranslated strings can be left blank.

I don't care if this is translate, by the way. It would be such a rare occurrence that a user would see this.

Reply:
I though it was an easter egg and some string of a game :smile: I'll remove it

Reply:
It's similar to an Easter egg, I guess. If a user tries giving '/' as an argument to rmw, they'll see that message. :)

Thanks for working on this and correcting previous mistakes. I'll probably finish reviewing it tomorrow and merge it.